### DISCUSSION TITLE
打擊 Deepfake 換臉影片，Facebook 透過台灣官方帳號發聲

### DISCUSSION THREAD

**Discussion by 239216** [1101-GS4719-C/Python-Community-GS4719-C#125] [General]
Facebook 19 日透過台灣官方帳號發文強調，根據社群守則規範，Facebook 用戶不得發布用 Deepfake 深度偽造技術建立的以假亂真影片，為首家向台灣社會積極表態的社群平台。

Facebook表示，先前已投入新台幣3億元獎金舉辦偵測Deepfake挑戰賽，持續精進辨識技術與工具。

**Comment by HSU-YU-JEN** [comment]
感覺Facebook也開始篩選平台上面的資訊了!!

**Comment by chun1203** [comment]
Deepfake深度偽造的技術感覺已經相當成熟，有時候甚至連人都無法辨識出真假，要用機器偵測出Deepfake感覺相當困難。

**Comment by Isongzhe** [comment]
感覺針對網路犯罪的法條要趕快跟上，才能避免更多這種事發生

**Comment by 109302003** [comment]
以前有看過一些深偽影片（合成韓劇），那時還不知道什麼是深偽，看影片的留言才知道是利用深偽合成，當時嚇到因為臉部角度、說話唇形都看不出有破綻，需有相關的法律規範才能避免非法行為造成傷害。
